### ISSUE TITLE
Sprint 107 Summary

### ISSUE THREAD

**Issue by AnthonyDGreen** [dotnet/roslyn#14378]
# Visual Basic and C# Compiler Team Accomplishments
- Bugs, Bugs, more Bugs.
- Debugger visualization for tuples (in progress).
- Consuming ref returns in VB
- Support for new features in scripting contexts.
- VB tuples catching up and metadata interop.
# Visual F# Tools Team Accomplishments
- Lots of work on Roslyn IDE integration
  - Completion added
  - Colourization added
  - Almost ready to merge Yay !!!!!! 
  - Many …. many … many [New VS Installer] bugs
- Debugging extravaganza
  - Embedded Portable PDBs
  - Embedded Source in portable PDBs
  - Fix: Nasty Stack Overflow in Full PDBs
- OSS
  - Moved over to squash and merge policy
  - 7 Authors – (3 MS – 4 External)
  - 72 commits (46 to master) 
  - 30 merged PRs
  - 14 proposed PRs
# Interactive Execution Team Accomplishments
- Lots of bug fixes and testing towards Preview 5 release
- Sending Preview 5 bits to select internal and external customers today for feedback
- Got Live Unit Testing working with VS 2017 - need to push changes to LUT repo
- Reliability bug fixes in Edit-and-Continue for RC
# Infrastructure Team Update
- Moved lab/ml to VSTS CI launcher - We will now have 4+ builds a day with full DDRIT [Integration Tests] and RPS [Performance Tests] data
- Integration tests on Dev15 made significant progress; most passing on [New VS Installer] builds

![image](https://cloud.githubusercontent.com/assets/10539978/19208635/6097c94a-8cb1-11e6-9f94-f9b04c86d9f1.png)
- Added new role to infrastructure SWAT rotation:
  - Mantis: Ty
  - **NEW** Grasshopper: Shyam
  - Viper: Fred
  - Tiger: Vlad
  - Crane: Tom
# Project System & Analysis Team Accomplishments
- Project System
  - Cross-targeting: build from commandline and VS
  - Move NuGet dependencies to PackageReference.
  - Contained Language support for Razor.
  - Startup project chooser
  - Cross-targeting: Combined IntelliSense
  - NuGet background restore
  - APIs for Package Manager UI.
  - Debug prop page with UI for Launch profiles.
- Analyzers
  - Perf tuning of out-of-process.
# IDE Team Accomplishments
- Moved IDE services out-of-process (needs dogfooding)
  - Find All References
  - Navigate To
  - Add Package/Reference
- More style options
  - Object initializers
  - Throw expressions
  - Conditional Access delegate invokes
- Shipped initial DPL [deferred project load] support in Preview 5
- .editorconfig support for basic properties

**Comment by hadibrais** [comment]
**TL;DR**

![Bugs](http://images.memes.com/meme/1128865)

**Comment by MgSam** [comment]
Looking forward to finding out what [Redacted] is.

**Comment by adamralph** [comment]
> Support for new features in scripting contexts.

That sounds good, but can we get any details?

**Comment by jmarolf** [comment]
@adamralph I believe this was explicitly referring to new language features such as tuples being supported in the interactive window.
